api_backend.yaml

```diff
@@ -261,6 +261,54 @@ paths:
           description: There was an error in retrieving the message.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/messages/{id}/message-status":
+    x-swagger-router-controller: MessagesController          
+    put:
+      operationId: upsertMessageStatusAttributes
+      summary: UpsertMessageStatusAttributes
+      description: Updates the status of a message with attributes
+      parameters:
+        - name: id
+          in: path
+          type: string
+          required: true
+          description: The ID of the message.
+        - name: body
+          in: body
+          schema:
+            $ref: "#/definitions/MessageStatusChange"
+          required: true
+          x-examples:
+            application/json: |
+              change_type: "bulk",
+              is_archived: true,
+              is_read: true
```

Comment:
Can this property always set with any boolean value? I remember that a read message cannot be set to unread. Should be a specific status code for the forbidden operation (403 Forbidden / 409 Conflict).

Reply:
Sorry, I see now that the type accepted for the is_read prop is only `true` value 😅